client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-large-preview.tsx

```diff
@@ -85,7 +84,6 @@ const PatternLargePreview = ( {
 			>
 				{ viewportHeight && (
 					<PatternRenderer
-						key={ device }
```

Comment:
@miksansegundo Do you remember why we use the device as a key to force rendering?

Reply:
Yes, it was introduced in https://github.com/Automattic/wp-calypso/pull/76132/files#r1175185900.

I remember this is for the case of patterns like Link In Bio with `min-height: 100vh` because when the device viewport changes the patterns must be re-rendered to calculate their content height.

However, we don't have Link In Bio patterns in the assembler, and that logic was cleaned recently in https://github.com/Automattic/wp-calypso/pull/81310

We can probably remove it. Let me test it again.

Reply:
Nice 👍

Reply:
The issue with `min-height: 100vh` and the scrolling is gone thanks to the refactoring. 

https://github.com/Automattic/wp-calypso/assets/1881481/601bca19-9346-4aef-ab87-6075fa14e3be

We could remove that key but then the animation won't look good. See this Intro pattern dancing 😆 when switching viewports.

https://github.com/Automattic/wp-calypso/assets/1881481/a889290f-39b5-4d81-a5d5-6a8f971258af

Reply:
Yes, it's dancing 😆 But the current behavior is not good either as all patterns become blank due to the key changes. Anyway, I'll revert the change and we can improve the experience later